improver_tests/utilities/cube_manipulation/test_collapse_realizations.py

```diff
@@ -32,18 +32,71 @@
 Unit tests for the function collapsed.
 """
 
+import unittest
```

Comment:
I know that you've haven't changed the docstring for this file, but could it be updated from `collapsed` to `collapse_realizations`?

Reply:
Done